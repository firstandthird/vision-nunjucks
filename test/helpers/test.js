'use strict';
module.exports = function(str) {
  return `${str} from test`;
};
