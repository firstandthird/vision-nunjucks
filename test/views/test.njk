{% extends "base.njk" %}

{% block content %}
  <p>hi</p>
{% endblock %}
