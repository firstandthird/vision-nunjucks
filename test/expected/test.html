<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>hi from nunjucks</title>
  </head>
  <body>
    
  <p>hi from test</p>

  </body>
</html>
